.github/workflows/runTestSinglePython.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         pip install flake8 pytest
         pip install pytest-cov
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f docs/requirements.txt ]; then pip install -r docs/requirements.txt; fi
```

Comment:
Is this really needed? As the test is not building the docs... and readthedocs knows about the docs/req...  anyways...

Reply:
I havent tried without but if you say reads the docs knows about it, I remove it

Reply:
If readthedocs does not automatically find it, we can point to it in the Advanced settings of the avaframe project on rtd...